src/pages/workspace/WorkspaceSettingsPage.js

```diff
@@ -138,8 +123,8 @@ class WorkspaceSettingsPage extends React.Component {
                                     fallbackIcon={Expensicons.FallbackWorkspaceAvatar}
                                     style={[styles.mb3]}
                                     anchorPosition={{top: 172, right: 18}}
-                                    isUsingDefaultAvatar={!this.state.previewAvatarURL}
-                                    onImageSelected={this.uploadAvatar}
+                                    isUsingDefaultAvatar={!this.props.policy.avatar}
```

Comment:
Probably should use `lodashGet` here:

```suggestion
                                    isUsingDefaultAvatar={!lodashGet(this.props.policy, 'avatar', null)}
```